modules/serverless-open-telemetry.adoc

```diff
@@ -0,0 +1,149 @@
+// Module is included in the following assemblies:
+//
+// * serverless/serverless-tracing.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-open-telemetry_{context}"]
+= Using {DTProductName} to enable distributed tracing
+
+{DTProductName} is made up of several components that work together to collect, store, and display tracing data. You can use {DTProductName} with {ServerlessProductName} to monitor and troubleshoot serverless applications.
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+* You have not yet installed the {ServerlessOperatorName} and Knative Serving. These must be installed after the {DTProductName} installation.
+* You have installed {DTProductName} by following the {product-title} "Installing distributed tracing" documentation.
+* Install the OpenShift (`oc`) CLI.
```

Comment:
This bullet has again a different style than the others: "Install ... vs. You have installed"